native/src/main/java/io/ballerina/stdlib/http/api/HttpUtil.java

```diff
@@ -1845,6 +1848,9 @@ private static void evaluateProtocolField(BMap<BString, Object> protocol,
         if (!sslProtocol.isBlank()) {
             sslConfiguration.setSSLProtocol(sslProtocol);
         }
+        if (System.getProperty(JAVA_CONFIG_TLS_NAMED_GROUPS).isBlank()) {
```

Comment:
We have to do a null check before checking `isBlank()`. Also it should be enough to check only for null property right? Why do we need to check whether it is blank?